etc/tempesta_fw.conf

```diff
@@ -259,8 +259,18 @@
 #       The value of the "Host" header field.
 #   - hdr_conn
 #       The value of the "Connection" header field.
+#   - hdr_ctype
```

Comment:
Not the issue of this PR, but it seems that all that `hdr_ctype`, `hdr_agent` and Co is not very user friendly. In the same time we have `hdr_raw`. It would better to provide single directive for all headers:
```
http_chain NAME {
	[ hdr "HDR_NAME" == (!=) ARG ] -> ACTION [ = VAL];
	...
}
```
Like it's done for `hdr_raw` now. And give a notice that comparing of the `Content-Type`, `User-Agent`, .... headers is accelerated, while processing of other headers is slow.

We have `tfw_http_msg_req_spec_hid()` function which can convert header name to it's internal representation in spec headers. So i believe that the change would be easy to implement.

Reply:
I created issue #1033